spec/features/admin_spec.rb

```diff
@@ -0,0 +1,115 @@
+require 'rails_helper'
+
+describe 'admin', type: :feature do
+
+  let(:admin) { User.create(username: 'admin', password: 'password',
+                            password_confirmation: 'password', first_name: 'Justin',
+                            last_name: 'Bieber', email: 'asdf@gmail.com',
+                            role: 1) }
+  let(:category) { Fabricate(:category) }
+  let(:category2) { Fabricate(:category) }
+  let(:item) { item = Fabricate.build(:item)
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.